src/components/UserInfo.js

```diff
@@ -1,38 +1,58 @@
-/**
- * "The UserInfo class is responsible for rendering information about the user on the page."
- */
 export default class UserInfo{
   #nameElement;
   #descriptionElement;
+  #avatarElement;
   #name;
   #description;
+  #avatar;
+
+/**
+ * Manage getting and setting information in profile section of page
+ * @param {object} elements {nameElement, descriptionElement, avatarElement} elements containing relevant profile information
+ * @param {object} userInfo {name, about, avatar} strings of user information to popuplate profile information on page
+ */
+  constructor(elements, userInfo){
+    //elements for name, description, and avatar
+    this.#nameElement = elements.nameElement;
+    this.#descriptionElement = elements.descriptionElement;
+    this.#avatarElement = elements.avatarElement;
+
+    //store name, description, and avatar in object
+    this.#name = userInfo.name;
```

Comment:
Needs correction: There is no need to save the contents in the class instance, you can find it via the dom methods